bin/kn

```diff
@@ -188,8 +202,11 @@ kubetoken)
   ;;
 
 # KubeNow-defined commands that need PWD validation
-apply | destroy | scale | kubectl | helm | ssh)
+apply | destroy | provision | scale | kubectl | helm | ssh)
   validate_pwd
+  # get provisioner image from tfvar-file
+  read_provisioner_image_var
```

Comment:
Part of the possibility to have different provisioners image depenting on if you are on a development branch.